R/read10xVisium.R

```diff
@@ -92,6 +92,8 @@ read10xVisium <- function(samples="",
     images="lowres",
     load=TRUE) {
     
+    .Deprecated("VisiumIO::TENxVisium(List)")
```

Comment:
I don't know how .Deprecated works, can we add a personalized comment to suggest people to look into VisiumIO package?

Reply:
If you type the above command in R, you’ll see that .Deprecate already gives a specific message suggesting using VisiumIO::TENxVisium(List); we can provide a custom version, though it wouldn’t be much different I think, but let me know what you think.

Reply:
nice, thanks for the clarification

Reply:
Yes, I think the default message looks something like this, which should be ok:
```
Warning message:
'f_old' is deprecated.
Use 'f_new' instead.
See help("Deprecated")
```